package.json

```diff
@@ -13,6 +13,7 @@
     "immer": "^5.0.0",
     "mark.js": "^8.11.1",
     "mithril": "^2.0.4",
+    "readability": "file:../readability",
```

Comment:
You might be able to install this via the repo with `yarn add [git-repo]#[tag]`.

If not I'd suggest adding it as a gift submodule.